### ISSUE TITLE
Replace Signup Progress Img => Component

### ISSUE THREAD

**Issue by divyanshu-rawat** [Ignitus/Ignitus-client#876]
**Is your feature request related to a problem? Please describe.**
Currently, if you navigate to http://www.ignitus.org/flow/welcomeFlow & follow the flow by clicking on `Save & Continue` you will notice that a progress bar is updating but currently what we do is we show different images on each transition from one page to another for example. The transition from http://www.ignitus.org/flow/welcomeFlow => http://www.ignitus.org/flow/interestFlow updates the image from the attached first attached image to the second attached image.

**First Image**

![Screenshot 2020-06-26 at 23 12 05](https://user-images.githubusercontent.com/13464678/85901605-79bb8800-b802-11ea-9441-3da25018d597.png)

**Second Image**
![One](https://user-images.githubusercontent.com/13464678/85901499-4aa51680-b802-11ea-912d-bc9bf9583272.png)

**Describe the solution you'd like**
We have already created a progress bar component that can be found here http://www.ignitus.org/interface/progress, signup progress images should be replaced by this component.

**Describe alternatives you've considered**
N.A

**Additional context**
In Summary: Instead of showing different Images on each iteration its better to sue a component that can actually show the signup progress flow.
Progress component source [code](https://github.com/Ignitus/Ignitus-client/blob/master/src/ignitus-UserInterfaceBook/Components/Molecules/interfaceProgress/index.tsx).

Please let me know if you have any Questions regarding this issue we can have a conversation here or on slack.

**Comment by Dishebh** [comment]
@divyanshu-rawat this as well
